samples/testbot/test-bot/GitHubHttpTriggers.cs

```diff
@@ -0,0 +1,318 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace DFTestBot
+{
+    using System;
+    using System.IO;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Http.Extensions;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    public static class GitHubHttpTriggers
+    {
+        const string CommandPrefix = "/DFTest ";
+
+        static readonly string TestAppSubscriptionId = Environment.GetEnvironmentVariable("TEST_APP_SUBSCRIPTION_ID");
+
+        [FunctionName("GitHubComment")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "POST")] HttpRequest req,
+            [DurableClient] IDurableClient durableClient,
+            ILogger log)
+        {
+            log.LogInformation($"Received a webhook: {req.GetDisplayUrl()}");
+
+            if (!req.ContentType.StartsWith("application/json", StringComparison.OrdinalIgnoreCase))
+            {
+                return new BadRequestObjectResult("Expected application/json");
+            }
+
+            dynamic json;
```

Comment:
I don't love the use of `dynamic` if there is a defined schema. Could we define a POCO class that defines this payload so we can get typing here?